tidb-cloud/tidb-cloud-quickstart.md

```diff
@@ -41,6 +41,10 @@ To create a free Developer Tier cluster, take the following steps:
 
```

Comment:
After clicking **Create**, *the **Security Settings** dialog box will not be displayed. Users have to open it manually. Please update the steps here accordingly.   And the https://docs.pingcap.com/tidbcloud/create-tidb-cluster doc needs to be updated as well.